### PR TITLE
Add a tool list in readme file

### PR DESCRIPTION
**Pull request by JiyangLiuG**
@schaefi You could decline this if you think it is not necessary. The code is very well self explained.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -4,4 +4,14 @@ EBCL - NautilOS Repo Setup
 Provides `ebcl-repo-setup` and `ebc-sync` tools to initialize
 credentials protected NautilOS debian repositories from Artifactory.
 An environment file `~/.ebcl` is created such that other ebcl tools
-can also make use of it
+can also make use of it.
+
+Tool list:
```
Comment (isbm) [inline README.rst:9]: List of tools.

File: README.rst [hunk 2]
```diff
@@ -4,4 +4,14 @@ EBCL - NautilOS Repo Setup
...
+- ebcl-releases
+  Print available release dates
```
Comment (isbm) [inline README.rst:11]: - `ebcl-releases`
  Prints available release dates
- `ebcl-repo-setup`
  Setup tool for package repositories and cache updates
- `ebcl-sync`
  Package repository syncer
- `ebcl-ui`
  SDK setup tool

**Review by isbm — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (isbm) [reply README.rst]: I believe the syntax is:

```markdown
- `fixed list item`
  Explanation comment

- `another fixed list item`
  Another explanation comment
```

So it is rendered then:

> - `fixed list item`
  Explanation comment
>
> - `another fixed list item`
  Another explanation comment

**Review by isbm — CHANGES_REQUESTED** [review]
Commented

File: README.rst (hunk 2 above)
Comment (JiyangLiuG) [reply README.rst]: Hi @isbm , sorry, i missed this comment. 
The syntax you used is rendered as.

```
fixed list item Explanation comment
another fixed list item Another explanation comment
```

And i checked https://sublime-and-sphinx-guide.readthedocs.io/en/latest/, 
Lists shall be started with '#' or '*', but not '-'.

Why you use '' about the list item, you would like to achieve which effect? Interpreted text? https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#toc-entry-49

**Comment by JiyangLiuG** [comment]
> Commented

replied above.

**Comment by JiyangLiuG** [comment]
open for too long. Will be done in other pull request together.

**Pull request closed without merging**